WordPress/src/main/java/org/wordpress/android/ui/sitecreation/verticals/SiteCreationIntentsFragment.kt

```diff
@@ -58,9 +61,19 @@ class SiteCreationIntentsFragment : Fragment() {
         siteCreationIntentsTitlebar.appBarTitle.isInvisible = !isPhoneLandscape()
         siteCreationHeaderItem.title.setText(R.string.new_site_creation_intents_header_title)
         siteCreationHeaderItem.subtitle.setText(R.string.new_site_creation_intents_header_subtitle)
+        recyclerView.layoutManager = LinearLayoutManager(activity, RecyclerView.VERTICAL, false)
```

Comment:
**Open Question which should not influence when we merge the PR:**
Is there any pro/con to setting the layout manager programattically vs setting it from the xml with `app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"`*?

Asking because I only found recently that [it's possible to set it from XML](https://stackoverflow.com/a/35680716/4129245), and for me it seems nicer to do it from there 😄 

I quickly tested if it works the same by setting it via XML and it does 🤷‍♂️

**\*** I guess if we do it from XML we should also add `android:orientation="vertical"`

Reply:
Ah, good call! I'll use the declarative approach. :+1: